LongestPalindrome.swift

```diff
@@ -0,0 +1,34 @@
+// Time Complexity : (O)n
+// Space Complexity : (O)1
+// Did this code successfully run on Leetcode : No. Wrong answer 
```

Comment:
Please go thru this java code and try again.
public int longestPalindrome(String s) {
        if (s.length() < 2) return s.length(); 
        HashSet<Character> set = new HashSet<>();
        int ans = 0;
        for (char c : s.toCharArray()) {
            if (!set.contains(c)) set.add(c);
            else {
                ans += 2;
                set.remove(c);
            }
        }
        return set.isEmpty() ? ans : ans + 1;
    }